browser-id3-writer.d.ts

```diff
@@ -0,0 +1,242 @@
+declare module 'browser-id3-writer' {
+  export const enum SynchronizedLyricsType {
+    Other = 0,
+    Lyrics = 1,
+
+    /**
+     * Text transcription
+     */
+    TextTranscription = 2,
+
+    /**
+     * Movement/part name (e.g. "Adagio")
+     */
+    MovementPartName = 3,
+
+    /**
+     * Events (e.g. "Don Quijote enters the stage")
+     */
+    Events = 4,
+
+    /**
+     * Chord (e.g. "Bb F Fsus")
+     */
+    Chord = 5,
+
+    /**
+     * Trivia/'pop up' information
+     */
+    Trivia = 6,
+  }
+
+  export const enum SynchronizedLyricsTimestampFormat {
+    /**
+     * Absolute time, 32 bit sized, using MPEG frames as unit
+     */
+    Frames = 0x01,
+
+    /**
+     * Absolute time, 32 bit sized, using milliseconds as unit
+     */
+    Milliseconds = 0x02,
+  }
+
+  // id3.org uses hex values for these. I'm keeping them as hex for consistency
+  // with the documentation.
```

Comment:
why is then `enum SynchronizedLyricsType` not same format?

Reply:
Because the only options are `0` or `1`, I can't tell if the docs are using base 10 or base 16.  I don't mind slapping an `0x` in front of them. Do you think I should?

Reply:
I think it's better to have same style/format for all enums so please change SynchronizedLyricsType to hex, or other enums to base 10

Reply:
Corrected in f6a9d3f